src/Raven.Server/Dashboard/DatabasesInfoRetriever.cs

```diff
@@ -157,8 +157,12 @@ public static IEnumerable<AbstractDashboardNotification> FetchDatabasesInfo(Serv
                         };
                         trafficWatch.Items.Add(trafficWatchItem);
              
-                        databasesOngoingTasksInfo.Items.Add(GetOngoingTasksInfoItem(database, serverStore, context, out var ongoingTasksCount));
-                        
+                        var ongoingTasksInfoItem = GetOngoingTasksInfoItem(database, serverStore, context, out var ongoingTasksCount);
```

Comment:
wait you compute that everytime and if collectOngoingTasks is false, then we discard computations?

Reply:
looks like you are using this count (passed via out parameter)